de.metas.adempiere.adempiere/base/src/main/java/de/metas/security/model/interceptor/SecurityMainInterceptor.java

```diff
@@ -59,6 +59,7 @@ protected void registerInterceptors(final IModelValidationEngine engine, final I
 		// Role and included roles
 		engine.addModelValidator(de.metas.security.model.interceptor.AD_Role.instance, client);
 		engine.addModelValidator(de.metas.security.model.interceptor.AD_Role_Included.instance, client);
+		engine.addModelValidator(AD_Document_Action_Access.instance, client);
```

Comment:
FYI, you can also annotate the  model interceptor with `@Component`, then spring will do the magic on startup